edit(Fast Data): Real-Time Updater as a plugin (#655)
mia-platform/documentation: docs/getting-started/tutorials/fast-data/low-code.mdx

Before starting this tutorial, you need to be familiar with the concepts of [Fast Data architecture](/fast_data/architecture.md), including [Projections](/fast_data/configuration/projections.md), [Single View](/fast_data/configuration/single_views.md), and Kafka (including producers and consumers).
For the technical part of this tutorial, an up-and-running Kafka cluster is required.

## System of Records

A System of Records is a container of _Projections_, which is are architectural objects responsible for storing the data in a persistent storage, to have a fast and scalable way to decouple the data from the source system.

> We will use this projections to have an easy way to read a 1:1 copy of the data from the source systems, and later it will be used to aggregate the data from the source systems, using the Single View Creator.

Reviewer: ```suggestion
We will use these projections to have an easy way to read a 1:1 copy of the data from the source systems, and later they will be used to aggregate the data from the source systems, using the Single View Creator.
```

The architectural component responsible for creating and updating the projection is the _Real-Time Updater_ service.
To simplify, this microservice is a Kafka consumer that will consume the data from the source systems and will update the projection.

In this tutorial, we can imagine a simple delivery platform where the data are the registry and the order of the customers.
So, we will create a single System of Records with two projections (one for the registry and one for the order) attached to a single Real-Time Updater.

### Microservice Real-Time Updater

:::info
From version `11.7.0` of the Console, the _Real-Time Updater_ service must be created manually with the guide explained in this paragraph. 

> If you're using an older version, you can skip this paragraph since the service to be automatically created when saving the configuration after creating the System of Records.

Reviewer: ```suggestion
If you're using an older version, you can skip this paragraph since the service is automatically created when saving the configuration after creating the System of Records.
```
